styles/components/TextSection.module.scss

```diff
@@ -0,0 +1,25 @@
+.TextSection {
+  @apply grid grid-cols-2 gap-4 p-4;
+  // @apply bg-color-4;
+  @apply py-16;
+  @apply px-10;
+}
+
+.TextAndButton {
+  @apply col-span-1;
+}
+
+.TextSectionText {
+  @apply text-color-2;
+  @apply leading-relaxed;
+  @apply text-xl;
+  @apply pr-4;
+  @apply mb-4;
```

Comment:
Is there a benefit to having each rule on it's own line? Or is it worth saving lines by grouping rules (ie @apply {all of the colouring rules} @apply {all of the padding/margins/gaps}

Reply:
I mean no real advantage other than saving a few lines to doing it together. Personally I just do whatever makes sense to me in the moment. Changed it tho